### DISCUSSION TITLE
audio crackle

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#526] [Help]
Originally posted 2018-09-17 02:48:17.297000 by mashaffer in GrandOrgue discussion forum on SourceForge<br><br>Sorry about the accidental discharge.

I now have  GO running with good latency on Ubuntu but I am noticing an occasional crackle like a bad cable even at low volume. I don&#039;t have that problem with jOrgan. Is there some configuration issue I should look at?

**Comment by larspalo** [comment]
Originally posted 2018-09-22 08:16:54.886000 by [Martin Koegler](https://github.com/e9925248) in GrandOrgue discussion forum on SourceForge<br><br>Its usually a short overload of your system - you need to configure GO more conservative to provide more reserves for such load spikes. Other options: run GO with real-time priority and/or reduce background processes.

**Comment by GIP94100** [comment]
Hi,
i have the same issue on 2 Windows PCs (16 Gb RAM) : when launching GO, the audio sound crackles without even playing.
What do you mean by "more conservative" ?
Thank you

**Comment by oleg68** [comment]
@GIP94100

1. Does the cracle continue while GO is running after launch?
2. Please post here the screenshot of the `Settings->Audio` tab.

**Comment by GIP94100** [comment]
Hi @oleg68,
yes the crackle is there just after GO launch, but it is discrete. Then, when playing, it becomes louder.
Here is the screen capture 
![Settings-Audio](https://user-images.githubusercontent.com/100627821/156169935-dca0cb23-aa16-48a7-8d28-50ea836719cc.jpg)

**Comment by oleg68** [comment]
You have a very old version of GO. Please install the latest from https://github.com/GrandOrgue/grandorgue/releases and post the screenshot again

**Comment by GIP94100** [comment]
Hi,
i installed the new version.
I have the same issue. More precise description/symptoms (sorry if i don't use the correct terminology, i am not an organist and i am not english) :
- if no drawstop engaged, no crackle
- as soon as a drawstop is engaged, without playing, a reguler light crackle appears
- when playing a chord with a lot of notes on the manual, and with a pedal, the crackle is louder

Here is the screenshot :
![Settings - Audio 2](https://user-images.githubusercontent.com/100627821/156244816-921ce334-5c24-4556-a2fe-6c84a019545c.jpg)

**Comment by oleg68** [comment]
@GIP94100 Seems your system is overloaded

1. What CPU do you have? At what speed does it run (take a look at the performance page of the Windows task manager)
2. What sampleset are you using?
3. Do you have another applications running at the same time?
4. Post here the Option tab screenshot.
5. Did you enable the Impulse reverb?

**Comment by GIP94100** [comment]
Hi Oleg,
1. Intel i7-3610QM. It runs at between 2.8 and 3Ghz with just GO running on the PC. RAM = 16 Gb
2. GrandOrgue Demo v1
3. No
4.
![Options](https://user-images.githubusercontent.com/100627821/156427297-b44f9042-d293-4a16-b091-b4bf093fdfc9.jpg)
5. No, i don't think so. I don't find it in the menus ...

Thank you

**Reply by oleg68** [reply]
1. Try to decrease ``Nombre de coeurs de CPU`` to 8.
2. Try to change the audio device in the Audio tab to another options.

**Comment by GIP94100** [comment]
Hi Oleg,
1. much better when decreasing the number of cores to 8, but some little crackles persist
2. I connected a Bose SoundLink Mini II. The sound from Windows on the PC is correctly amplified on this device. So I configured it on GO, but never got an organ sound on the Bose, although I tried all the drivers (MME, WASAPI, etc ...)

Well, due to all these issues, i decided to try GrandOrgue on another PC, different from Asus : installed it on a Dell PC (Windows). Result : no more crackling :-) !  Great. So, i'll keep this PC to use GO.

Just 2 last questions :
a. it seems the volume spinner doesn't have any effect on the heard volume. I put 20 but it doesn't change the volume you hear. I have this behaviour on both PC : the Asus and the Dell
b. hox to reduce the small "lag" between a note that you play on the keyboard and the sound it produces ?

Thank you very much for all your help

**Reply by rousseldenis** [reply]
Following the specs of your CPU, it should be set to 4 cores I think.

For your delay problem, you need to check buffer setting (in my case 512 is better than 1024).
